PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingAnalytics.kt

```diff
@@ -0,0 +1,7 @@
+package com.braintreepayments.api;
+
+internal enum class PayPalMessagingAnalytics(val value: String) {
+    STARTED("paypal-messaging:create-view:started"),
+    FAILED("paypal-messaging:create-view:failed"),
+    SUCCEEDED("paypal-messaging:create-view:succeeded");
```

Comment:
```suggestion
    SUCCEEDED("paypal-messaging:create-view:succeeded")
```
Linter picked this up too. We also need to add a new line to the end of this file, then the checkmarks should go 🟢 .